package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "@ionic/prettier-config": "^2.0.0",
-    "@stencil/core": "^2.9.0",
```

Comment:
Should we update the `package-lock.json` files here (and in the other files) as well? `devDependencies` and `peerDependencies` still get written to the manifest when running `npm i`. I think this might get a little tricky with respect to my other comment on this line though 😢

Reply:
ah yeah that's probably a good idea 😅 

I think I missed it just because I was `lerna`-ing and I thought it would do that automatically!